### ISSUE TITLE
Name property is not provided for "Delete" button present besides to 'Project' heading: A11y_.NET 9 template_Project_Screen reader

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27429]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG4.1.2; #A11ySev3; #iOS; #SH_.NET9template_iOS_Jan2025; #Voiceover; #.NET Core; #Device; #screen reader; #NameRoleValue; #AINotInscope; #iphone;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2
Screen reader: Voiceover

### Repro Steps:

1. Turn on Voiceover and Install and open ".NET 9 template" app.
2. "Home Screen" will open. Swipe till Hamburger menu and double tap on it, Projects options will screen will open.
3. Swipe to any project and double tap to activate it.
4. Project screen will open. Swipe to "Delete" button.
5. Observe that Name property is not provided for "Delete" button present besides to 'Project' heading.

**Similar issue also observed in below scenario:
Issue 1: .NET 9 template_ Manage meta**

1. Turn on Voiceover and Install and open ".NET 9 template" app.
2. "Home Screen" will open. Swipe till Hamburger menu and double tap on it, swipe till Manage meta menu and double tap on it.
3. "Categories and tags" screen will appear. Swipe to all "Delete" buttons.
4. Observe that Name property is not provided for all the "Delete" button present in "Categories and tags" screen.

### Actual Result:
Name property is not provided for "Delete" button present besides to 'Project' heading.

### Expected Result:
Appropriate name property should be provided for "Delete" button present besides to 'Project' heading.

### User Impact:
Users who rely on screen reader won't be able to understand the purpose of the button if name property is not provided for it.

### Attachment:

https://github.com/user-attachments/assets/e3f8ee33-4b50-499a-9d76-7663091c0e63

**Issue 1:**

https://github.com/user-attachments/assets/bfff7ebc-c66d-4c0c-8fac-c5541bb24405

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by kubaflo** [comment]
HI @PoojaNamde it looks like this PR fixed the issue https://github.com/dotnet/maui-samples/pull/669

**Comment by PoojaNamde** [comment]
#Closed; #A11yFixed; #Regressed:02-16-26;

Verified the issue in latest environment. Issue is fixed hence closing the bug.

https://github.com/user-attachments/assets/bc2cc5aa-805b-46dc-93bb-0f3e15f3ffbd
